src/fmu/ensemble/ensemble.py

```diff
@@ -130,6 +137,10 @@ def __init__(
         if isinstance(runpathfile, pd.DataFrame) and not runpathfile.empty:
             count = self.add_from_runpathfile(runpathfile, runpathfilter)
 
+        if manifest:
+            # The _manifest variable is set using a property decorator
+            self.manifest = manifest
```

Comment:
Not sure how this works relative to the decorator, but should this be `self._manifest = manifest`? And how does this work if a yaml-file is given to the manifest argument? Will it ever be parsed, or will `self.manifest` just contain the str filename?